### DISCUSSION TITLE
Convert tasks into issues

### DISCUSSION THREAD

**Discussion by vinicius-santos-vs** [community/community#52176] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

I'm not being able to convert tasks into issues when I'm inside of an issue. Does anybody know if is that a bug or the feature was removed?

**Comment by BREN0sx** [comment]
It's possible that the feature to convert tasks into issues has been removed or disabled by the repository owner or administrator. However, it's also possible that this is a bug.

One thing you could try is to check if the feature is available on the repository's web interface by navigating to the "Issues" section and seeing if there's an option to convert tasks into issues. If the option is available there but not working when you're inside of an issue, then it might be a bug.

In that case, you can try reaching out to the repository owner or administrator to report the issue and see if they can help you with a solution. Alternatively, you could also try using a different browser or clearing your browser cache to see if that resolves the issue.

If the feature has indeed been removed or disabled, you could consider using a different tool or workflow to manage your tasks and issues. There are many task management and issue tracking tools available that might better suit your needs.

**Comment by navanjr** [comment]
I too am no longer seeing the "Convert to Issue" feature when hovering over a task tested using:
- [ ] Chrome 111
- [ ] Brave 1.49

**Comment by rileybroughten** [comment, marked as answer]
@vinicius-santos-vs @BREN0sx @navanjr - Can you confirm whether this has been resolved on your side? Looks like a similar issue was reported in this [discussion](https://github.com/orgs/community/discussions/51960).

**Reply by navanjr** [reply]
nope. still not working for us. 😢

**Reply by vinicius-santos-vs** [reply]
@rileybroughten - It has been resolved on my side now. Working properly!

**Reply by navanjr** [reply]
@rileybroughten It is working again for us this morning. Thanks!

**Reply by tapash32** [reply]
Very good project 👍

**Comment by eloir-vs** [comment]
it's back
![image](https://user-images.githubusercontent.com/109624375/230892119-ebb0a4bc-a424-44eb-b8ed-d2284b0f4cfd.png)

**Reply by tapash32** [reply]
Sui token unlock please
